test-fixtures/bundle-app-node.js

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env node
-import __module from 'module';import __path from 'path';import __url from 'url';const require = __module.createRequire(import.meta.url);const __dirname = __path.dirname(__url.fileURLToPath(import.meta.url));const __filename=new URL(import.meta.url).pathname
+/* Generated by Spago v0 */import __module from 'module';import __path from 'path';import __url from 'url';const require = __module.createRequire(import.meta.url);const __dirname = __path.dirname(__url.fileURLToPath(import.meta.url));const __filename=new URL(import.meta.url).pathname
```

Comment:
Unfortunately the comment cannot be on a separate line, because the banner cannot contain newlines, which is in turn because on Windows command-line parameters cannot contain newlines. 

This can be worked around by importing `esbuild` as a Node dependency and calling it directly from PureScript via JS API rather than via command line. But that seemed like too much work for such insignificant gain.

Reply:
We can't import esbuild as a node dependency, because we won't be able to bundle it in and distribute that: it's written in Go so it relies on funny JS packaging to be distributed. In any case, this change is fine as is 🙂

Reply:
Well, an alternative could be to write a JS file that imports and runs esbuild, and then run that file.